app/src/main/res/menu/search.xml

```diff
@@ -3,16 +3,16 @@
 <menu xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:android="http://schemas.android.com/apk/res/android">
 
-  <item
-    android:id="@+id/legacy_search"
-    android:title="@string/action_search"
-    android:icon="@drawable/action_search"
-    app:showAsAction="always" />
+    <item
```

Comment:
Add SearchView instead of custom implementation

Reply:
Good, but on a Pixel (trying a “small” screen), the row gets pushed to outside the screen. I wouldn't worry about this now. As it's design that I think could be pushed later in time. 👍 

If I click on a search history item, the search input is not filled with the text. Is that on purpose? But I think, again that we could improve this on a task later. 👍 

Here's a GIF for both things:

![click-search-history](https://user-images.githubusercontent.com/465723/53559703-ff65ef80-3b4a-11e9-9a04-9c9b9fe1cdad.gif)

Reply:
Suggestions are not working properly as of now. I'm working on fixing them.

Reply:
OK. Sorry, I still don't know which parts were inherited and so need fixes 👍

Reply:
The suggestions are fixed in latest changes on recyclerview branch. Once this PR is merged, we can go ahead with them.